mongodb/activity/queryDocument/README.md

```diff
@@ -0,0 +1,82 @@
+<!--
+title: MongoDB Query Document
+weight: 4622
+-->
+# MongoDB Query Document
+This activity allows you to query for one or more documents from a collection in MongoDb database based on a matching criteria entered as a JSON
+
+## Installation
+
+### Flogo CLI
+```bash
+flogo install github.com/project-flogo/datastore-contrib/mongodb/activity/queryDocument
+```
+
+## Configuration
+
+### Settings:
+| Name                   | Type       | Description
+| :---                   | :---       | :---    
+| mongoConnection        | connection | Choose a MongoDB connection from the drop down  - ***REQUIRED***
+| operation              | string     | The Query operation type (Find One Document or Find Many Documents) - ***REQUIRED***
+| collectionName         | string     | The collection to work on - ***REQUIRED***  
+
+### Input: 
+
+| Name               | Type   | Description
+| :---               | :---   | :---  
+| jsonDocument       | object | The matching critiera entered as a JSON
```

Comment:
"jsonDocument" shouldn't this be "criteria" ?

Reply:
modified as criteria